Build on Windows

mdpiper opened the pull request:
This PR builds the Fortran BMI specification on Windows.

<!--
Thank you for pull request.
Below are a few things we ask you kindly to self-check before getting a review. Remove checks that are not relevant.
-->
Checklist
* [x] Used a fork of the feedstock to propose changes
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

<!--
Please note any issues this fixes using [closing keywords]( https://help.github.com/articles/closing-issues-using-keywords/ ):
-->

<!--
Please add any other relevant info below:
-->
(Bumped build number -- version didn't change.)

mdpiper committed:
Build on Windows

mdpiper committed:
Build on Windows

mdpiper committed:
MNT: Re-rendered with conda-build 3.18.8, conda-smithy 3.4.1, and conda-forge-pinning 2019.08.22

mdpiper committed:
MNT: Re-rendered with conda-build 3.18.8, conda-smithy 3.4.1, and conda-forge-pinning 2019.08.22

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe```) and found it was in an excellent condition.

mdpiper committed:
Add build file for Windows

mdpiper committed:
Add build file for Windows

mdpiper committed:
Change install location

mdpiper committed:
Change install location

mdpiper committed:
Use VS 14 generator to build

mdpiper committed:
Use VS 14 generator to build

mdpiper committed:
Revert "Use VS 14 generator to build"

This reverts commit a5c57aee2bd83b9949a853fbbb74ce47beb25e0b.

mdpiper committed:
Revert "Use VS 14 generator to build"

This reverts commit a5c57aee2bd83b9949a853fbbb74ce47beb25e0b.

mdpiper committed:
Add CMAKE_PREFIX_PATH

mdpiper committed:
Add CMAKE_PREFIX_PATH

mdpiper committed:
Add install target

mdpiper committed:
Add install target

Commit:
updated v1.2.1

mdpiper committed:
Use new jinja variables to set mod file name correctly

mdpiper committed:
Test for installed static library

mdpiper committed:
Don't use parallel make

It's causing problems in the Azure build process (can't find the Fortran
module file bmif_1_2.mod).

mdpiper committed:
Merge pull request #3 from regro-cf-autotick-bot/1.2.1

bmi-fortran v1.2.1

mdpiper committed:
Merge branch 'mdpiper/build-on-windows' of github.com:mdpiper/bmi-fortran-feedstock into mdpiper/build-on-windows

mdpiper committed:
Use cxx compiler on Windows for linker

mdpiper committed:
Add tests for Windows

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I wanted to let you know that I linted all conda-recipes in your PR (```recipe```) and found some lint.

Here's what I've got...


For **recipe**:

 * Selectors are suggested to take a ``<two spaces>#<one space>[<expression>]`` form. See lines [26]

mdpiper commented:
Success in #4.
